### PR TITLE
Remove legacy filter, cloudusage and coverage. Change references to f…

### PR DESCRIPTION
**Pull request by Sean-Holcomb**
…ilter 21 to filter

## What does this PR change?
This PR removes dead code from OpenCOst

## Does this PR relate to any other PRs?
This Pr is a recreation of https://github.com/opencost/opencost/pull/2515

## How will this PR impact users?
* 

## Does this PR address any GitHub or Zendesk issues?
* Closes ...

## How was this PR tested?
* Test to make sure there was no dependencies on removed code

## Does this PR require changes to documentation?
* 

## Have you labeled this PR and its corresponding Issue as "next release" if it should be part of the next OpenCost release? If not, why not?
*

### CODE REVIEW

**Review by nik-kc — APPROVED** [review]
:100:

**Review by ameijer — APPROVED** [review]
LGTM, love to see it

File: core/pkg/opencost/bingen.go [hunk 1]
```diff
@@ -72,4 +70,4 @@ package opencost
 // @bingen:generate:CloudCostLabels
 // @bingen:end
 
-//go:generate bingen -package=opencost -version=17 -buffer=github.com/opencost/opencost/core/pkg/util
+//go:generate bingen -package=opencost -version=18 -buffer=github.com/opencost/opencost/core/pkg/util
```
Comment (mbolt35) [inline core/pkg/opencost/bingen.go:73]: This part, I am a bit unsure about... Are these types on fields of on any other types? 

```
// @bingen:generate:Coverage
// @bingen:generate:CoverageSet
```

Comment (mbolt35) [reply core/pkg/opencost/bingen.go:73]: Looks like no. I _think_ this is fine. Deletion is a tricky though.

**Pull request merged**